clients/js/src/types.ts

```diff
@@ -72,3 +72,7 @@ export type AddResponse = {
 }
 
 export type CollectionMetadata = Record<string, unknown>;
+
+export type ConfigOptions = {
+  options?: RequestInit;
```

Comment:
does this type need to be imported?

Reply:
maybe we can also put a link in the comment here of what the valid `RequestInit` options are? I think that would be useful for future devs